metrics/prometheus.go

```diff
@@ -1746,6 +1746,54 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 		})
 	}
 
+	if includedMetrics.Has(container.PressureMetrics) {
+		c.containerMetrics = append(c.containerMetrics, []containerMetric{
+			{
+				name:      "container_pressure_cpu_stalled_seconds_total",
```

Comment:
According to [the docs](https://facebookmicrosites.github.io/cgroup2/docs/pressure-metrics.html), there is no "Full" for CPU. Just "Some". We can drop this metiric as it will always be zero.

Reply:
Great catch.

Reply:
this is only true for the top level cgroup, but not for children AFAIU https://docs.kernel.org/accounting/psi.html#pressure-interface